### PR TITLE
remove answer notification code

### PR DESCRIPTION
**Pull request by ViditChitkara**
Part of #4094

### CODE REVIEW

**Comment by jywarren** [comment]
Looks good; we resolved an upstream test failure today so perhaps this just needs a rebase? Not sure, though! Thanks!!!!

**Comment by ViditChitkara** [comment]
@publiclab/reviewers , @jywarren I think this is ready to merge.

File: app/controllers/answer_like_controller.rb [hunk 1]
```diff
@@ -11,8 +11,7 @@ def likes
       AnswerSelection.set_likes(current_user.uid, @answer.id, false)
     else
       AnswerSelection.set_likes(current_user.uid, @answer.id, true)
-      user = User.find(current_user.uid)
-      AnswerMailer.notify_answer_like(user, @answer).deliver_later
+      User.find(current_user.uid)
```
Comment (jainaman224) [inline app/controllers/answer_like_controller.rb]: I think this is not required. We already have user as `current_user`.
😄

Comment (ViditChitkara) [reply app/controllers/answer_like_controller.rb]: Yeah, I think that line didn't make any sense. Thanks for pointing out!!

**Comment by ViditChitkara** [comment]
> Looks good but is this supposed to be there? Just checking!

Woah, I think that was added by mistake. Removed it.

**Comment by jywarren** [comment]
Looks good!!!! Thank you!!!

**Pull request merged**